controllers/useraccount/useraccount_controller_test.go

```diff
@@ -65,8 +65,9 @@ func TestReconcile(t *testing.T) {
 			Name: ToIdentityName(userAcc.Spec.UserID, config.Auth().Idp()),
 			UID:  types.UID(userAcc.Name + "identity"),
 			Labels: map[string]string{
-				"toolchain.dev.openshift.com/owner": username,
-				toolchainv1alpha1.ProviderLabelKey:  toolchainv1alpha1.ProviderLabelValue,
+				toolchainv1alpha1.OwnerLabelKey:    username,
+				toolchainv1alpha1.SpaceLabelKey:    username,
+				toolchainv1alpha1.ProviderLabelKey: toolchainv1alpha1.ProviderLabelValue,
```

Comment:
we don't need the changes in the `useraccount_controller_test.go` file since we won't touch the UserAccount logic for now, right?